### ISSUE TITLE
Crash when auto save / load another world

### ISSUE THREAD

**Issue by Skyamoeba** [Panakotta00/FicsIt-Networks#153]
Game crashes when autosaving / loading in another save file.

**Comment by Skyamoeba** [comment]
[SMMDebug_2021-8-21_9-38-22.zip](https://github.com/Panakotta00/FicsIt-Networks/files/7025338/SMMDebug_2021-8-21_9-38-22.zip)

**Comment by Panakotta00** [comment]
Could be related to #152

Done anything within a drive folder like that?

Not just creating file but even kust changing a file might cause it...

Or does that crash happen anyway?

**Comment by CageStooge** [comment]
This happens to me from time to time without making any changes to files on the drive. I set my auto save down to 1 minute to see if I can find some common thing I am doing or have recently done that may cause it.

**Comment by CageStooge** [comment]
I did just notice the file systems are saved at <user>\AppData\Local\FactoryGame\Saved\SaveGames\Computers
Our save games are located in a different location, so the same AppData\Local\FactoryGame\Saved\SaveGames\Computers would be the same for different worlds. Is it possible the mod has some type of issue related to that? I would think the use of the UUID would limit that, but it was one thing I had noticed. In part because I don't even know what UUID drives are for which save game at this point.
